src/main/kotlin/no/nav/syfo/narmesteLederApi/NarmesteLederClient.kt

```diff
@@ -17,7 +17,7 @@ class NarmesteLederClient(
 ) {
     suspend fun hentNarmesteLederFraSyfoserviceStrangler(nlAktorId: String): List<NarmesteLederRelasjon> {
         val accessToken = accessTokenClient.hentAccessToken(resourceId)
-        return client.get<List<NarmesteLeder>>("$endpointUrl/api/$nlAktorId/narmesteleder") {
+        return client.get<List<NarmesteLeder>>("$endpointUrl/api/arbeidsgiver/$nlAktorId/narmesteleder") {
```

Comment:
Trenger vi `/api`?

Reply:
☝️

Reply:
Vi gjør vel strengt tatt ikke det, det har vel bare blitt et mønster siden vi bruker det i de andre appene våre. Dette er jo mot syfoservicestrangler da så det krever at vi endrer på den og. Jeg vet ikke hva rasjonalet var for å bruke /api på roten i utgangspunktet :p